index.html

```diff
@@ -0,0 +1,1811 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+
+    <!-- Bootstrap CSS -->
+    <script
```

Comment:
- [ ] Here, kindly delete the commented codes. If you need to review a previous version of your code, you can make use of git history.